javascript/package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@rails/actioncable": ">= 6.0",
     "cable_ready": ">= 4.1.2",
+    "form-serialize": "^0.7.2",
```

Comment:
I wonder if we should loosen this dependency up a bit. `"form-serialize": ">= 0.7.2"` Thoughts?